platform/README.adoc

```diff
@@ -23,6 +23,21 @@ Requirements:
 . Build client
 +
 ```shell
+yarn
+```
++
+
+. Generate resolvers, and client side queries
++
```

Comment:
Wrong place. This is for people who want to run the platform

Reply:
Where do you see this command? Should it be in a `CONTRIBUTING` file?

Reply:
Contributing is more for contributing to our project. but this will be actually source code for the people so I guess we need new section here but not in running your app section as all should be compiled already. 

Then we can remove individual client and client-admin readmes.

Reply:
Yes, makes more sense.